packages/block-library/src/cover/edit.js

```diff
@@ -212,6 +213,8 @@ class CoverEdit extends Component {
 			noticeUI,
 			overlayColor,
 			setOverlayColor,
+			onResizeStart,
```

Comment:
there is too many `onResizeStart` & `onResizeStop` functions in this file I'm surprised `eslint/no-shadow` didn't stop it, while this is not a problem, can we possibly name it to something else? just to avoid any confusion? and better explain the goal of it? would something like `deselectBlock` & `selectBlock`

Reply:
Thanks, that's a good point. Will address in a bit.

Reply:
Fixed now in the most recent commit. As for the variable names, they were in different components and passed down from one to the other, so not really a shadowing issue.

However, it made me realize that the passed down `onResizeStart` prop wasn't actually used in `ResizableBox`. Renamed the variable names now to make it more clear too.